dissect/target/plugins/apps/other/env.py

```diff
@@ -24,8 +24,8 @@ def check_compatible(self) -> None:
         pass
 
     @export(record=EnvironmentFileRecord)
-    @arg("--env-path", help="path to scan environment files in", required=True)
-    @arg("--extension", help="extension of files to scan", default="env")
+    @arg("--env-path", type=str, help="path to scan environment files in", required=True)
+    @arg("--extension", type=str, help="extension of files to scan", default="env")
```

Comment:
`str` is also already the default type: https://docs.python.org/3/library/argparse.html#type